src/RoveComm/RoveCommTCP.cpp

```diff
@@ -0,0 +1,498 @@
+/******************************************************************************
+ * @brief RoveComm TCP class implementation.
+ *
+ * @file RoveCommTCP.cpp
+ * @author Eli Byrd (edbgkk@mst.edu)
+ * @date 2024-02-07
+ *
+ * @copyright Copyright Mars Rover Design Team 2024 - All Rights Reserved
+ ******************************************************************************/
+
+#include "RoveCommTCP.h"
+#include "RoveCommPacket.h"
+#include <arpa/inet.h>
```

Comment:
Put implicit include in 
/// \cond
#include <array>

/// \endcond
to avoid them being included in doxygen graph.